doc/en/components/inputs/combo/templates.md

```diff
@@ -21,6 +21,7 @@ The {ProductName} ComboBox component allows defining custom templates for differ
 
 The `itemTemplate` is a custom template that if defined should be used when rendering items in the list of options.
 
+<!-- WebComponents -->
```

Comment:
Please also add the item template and the group header template snippets for React as currently they seem to be missing